src/app/pages/vehicle.cpp

```diff
@@ -1,29 +1,41 @@
 #include <QPalette>
```

Comment:
actually in this whole file check to make sure anything font related matches whats in develop (i see a few deviations from it)